AirCasting/SearchAndFollow/SearchAndFollowMap/SearchAndFollowMap.swift

```diff
@@ -51,16 +60,18 @@ struct SearchAndFollowMap: UIViewRepresentable {
     }
     
     func updateUIView(_ uiView: GMSMapView, context: Context) {
-        if !showRedoButton {
+        if selectedPointerID != context.coordinator.pointerIdHolder {
+            context.coordinator.pointerIdHolder = selectedPointerID
+            placeDots(uiView, context: context)
+        }
+        if !showSearchAgainButton {
             placeDots(uiView, context: context)
```

Comment:
So this line is triggered whenever the user selects a new marker, so placeDots is called twice. I think we can avoid this by putting this line in an else if

Reply:
That was it, totally amazing. Thx a lot!